pages/index.js

```diff
@@ -29,23 +31,30 @@ export async function getServerSideProps(context) {
   }
 
   let total = {};
-  try {
-    const res = await fetch(
-      `${process.env.NEXT_PUBLIC_BASE_URL}/api/statistics/totals`
-    );
-    total = await res.json();
-  } catch (e) {
-    logger.error(e, "ERROR total stats not found ");
+  const {statusCode, totalStats} = await getTotalStats();
+  if (statusCode !== 200) {
+    logger.error(totalStats.error, "ERROR total stats not found ");
+    total = {
+      views: 0,
+      clicks: 0,
+      users:  0,
+      active:  0,
+    }
+  } else {
+    total = totalStats;
   }
 
   let today = {};
-  try {
-    const res = await fetch(
-      `${process.env.NEXT_PUBLIC_BASE_URL}/api/statistics/today`
-    );
-    today = await res.json();
-  } catch (e) {
-    logger.error(e, "ERROR today stats not found");
+  const {statusCode: status, todayStats} = await getTodayStats();
+  if (status !== 200) {
+    logger.error(todayStats.error, "ERROR today stats not found");
+    today = {
+      views: 0,
+      clicks: 0,
+      users: 0,
```

Comment:
I don't think these are needed as the api should always return a result event if `0`